dotCMS/src/integration-test/java/com/dotcms/experiments/business/ExperimentAPIImpIT.java

```diff
@@ -204,4 +244,618 @@ public void testADeleteOriginalVariant_shouldFail() throws DotDataException, Dot
         APILocator.getExperimentsAPI().deleteVariant(newExperiment.id().orElse(""), originalVariant.id(),
                 APILocator.systemUser());
     }
+
+    /**
+     * Method to test: {@link ExperimentsAPIImpl#getEvents(Experiment)}
+     * When: Call the methods with a Experiment with 4 Session each one with several
```

Comment:
Don't understand this text

Reply:
done https://github.com/dotCMS/core/pull/24034/commits/cb564daa0c443e886c8931149f0ef0a62935fc0f#diff-25dea0f580007d55e9704544452fcf0833029a2294fb814f4dddfb1bb2e4ba9bR249